app/upgrades.go

```diff
@@ -150,23 +157,56 @@ func FundCommunityPoolModule(
 	distKeeper.SetFeePool(ctx, feePool)
 }
 
-// AddNewPermissionsToStabilityCommittee adds the following permissions to the committee with the passed in id:
+// UpdateStabilityCommitteePermissions updates the following permissions to the
+// committee with the passed in id:
+//
+// Add:
 // - CommunityCDPRepayDebtPermission
 // - CommunityPoolLendWithdrawPermission
 // - CommunityCDPWithdrawCollateralPermission
-func AddNewPermissionsToStabilityCommittee(ctx sdk.Context, committeeKeeper committeekeeper.Keeper, committeeId uint64) {
+//
+// Remove:
+// - AllowedParamsChange for x/evm subspace (no longer stored in params)
+func UpdateStabilityCommitteePermissions(ctx sdk.Context, committeeKeeper committeekeeper.Keeper, committeeId uint64) {
 	// get committee
 	comm, found := committeeKeeper.GetCommittee(ctx, committeeId)
 	if !found {
 		panic(fmt.Sprintf("expected to find committee with id %d but found none", committeeId))
 	}
-	// set new permissions
 	perms := comm.GetPermissions()
+
+	// Remove the x/evm param permissions
+	for i, permission := range perms {
```

Comment:
We should only make this update for mainnet or the `v0.22.0` upgrade.  Since we already migrated testnet, this would create a binary that can't be synced from the upgrade block.

Reply:
We'll have to issue a gov proposal on the public testnet